### DISCUSSION TITLE
Add the ability to append another video file to append with auto-editing.

### DISCUSSION THREAD

**Discussion by MilkAndBanana1** [WyattBlue/auto-editor#589] [2. Feature Ideas]
I have found that I would be working with 2 or 3 videos in which only one of the video files will be used in determing the edits. It would be nice to have an additional parameter where you can add video files that will only be affected by whatever edits will be done in the input parameter. 

```shell
auto-editor.exe gameplay.mp4 --append camera.mp4,chat_replays/chat.mp4 --export resolve 
``` 
This will be a real life example of how I would use this feature. So the video will be edited with the _gameplay.mp4_ file's audio, and all the cuts done to that file will also be done to the _camera.mp4_ and _chat.mp4_. Once exported, it is then opened in say Davici Resolve, will show up as seperate video and audio tracks (making them linked like the original video would be great as well).

**Comment by R1ON** [comment]
Bump, good idea!

**Comment by WyattBlue** [comment]
Files can have commas so this syntax would be better:

```
auto-editor game.mp4 --append camera.mp4 chat.mp4 --export resolve 
```

Setting options could be done like this:
```
auto-editor game.mp4 --append camera.mp4:offset=1sec,link=#f chat.mp4:link=#f --export resolve
```
 `offset` would change the clip's starting offset. `link` set to true would link video/audio clips to main.
